pkg/test/verify/cluster.go

```diff
@@ -162,10 +169,12 @@ func UpdateToolchainCluster(t *testing.T, functionToVerify FunctionToVerify) {
 	defer gock.Off()
 	statusTrue := test.NewClusterStatus(toolchainv1alpha1.ToolchainClusterReady, corev1.ConditionTrue)
 	toolchainCluster1, sec1 := test.NewToolchainCluster("east", "secret1", statusTrue,
-		Labels("", "", test.NameMember))
+		Labels("", test.NameMember))
+	toolchainCluster1.Labels["namespace"] = test.HostOperatorNs
```

Comment:
the `Labels` function already set the `namespace` label, so you can pass it there right away 
```suggestion
		Labels(test.HostOperatorNs, test.NameMember))
```